tests/copy_elision.cpp

```diff
@@ -29,10 +29,11 @@ std::size_t ArrayCtr::default_ = 0;
 
 ArrayCtr doubleArrayMaker()
 {
-  ArrayCtr result;
+  constexpr int size = 10;
+  ArrayCtr      result;
   // Just a random loop - if nothing was done to the array before returning,
   // the compiler might be able to do an unnamed returned value optimimization
-  for (auto i = 0; i < 10; i++) {
+  for (auto i = 0; i < size; i++) {
```

Comment:
Is this something clang-tidy complained about?

Reply:
Yeah, it considers this a "magic number" - though there's still a lot more floating around

Reply:
I agree with the idea of removing magic numbers in general, but it seems like there's a lot of cases where they are fine (or even preferable).

Reply:
I figured it was the case just making sure.  Is it possible to turn off magic numbers on the test directory? (don't put too much effort into this)

Reply:
Yes, I've set it up as two different check targets with the magic number warnings only enabled for the stuff in `src/`.